index.html

```diff
@@ -45,6 +45,7 @@ <h2><code>&lt;a&gt;</code></h2>
           <a class="whatsapp"></a>
           <a class="copy-url"></a>
           <a class="web-share"></a>
+          <a class="tumblr"></a>
```

Comment:
Please, put Tumblr between Telegram and Twitter to maintain alphabetical order everywhere where this applies (here, later in the HTML as well as in the CSS)